packages/styletron-engine-atomic/src/client/client.js

```diff
@@ -82,9 +83,32 @@ class StyletronClient implements StandardEngine {
     const onNewStyle = (cache, id, value) => {
       const {pseudo, block} = value;
       const sheet: CSSStyleSheet = (this.styleElements[cache.key].sheet: any);
-      const rule = styleBlockToRule(atomicSelector(id, pseudo), block);
+      const selector = atomicSelector(id, pseudo);
+      const rule = styleBlockToRule(selector, block);
       try {
         sheet.insertRule(rule, sheet.cssRules.length);
+        if (__BROWSER__ && window.__STYLETRON_DEVTOOLS__) {
```

Comment:
it would be nice to extract this and write some tests

Reply:
Will look into it 👍